### ISSUE TITLE
[TW-1809] task add description:ide fails

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1830]
_Sergey Trofimov on 2016-05-26T08:51:07Z says:_ 

 ```
~/devel/task 2.6.0
❯ task add description:ide
The value is not an expression.
```

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-05-26T08:51:07Z
Modified: 2017-01-24T17:09:59Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2016-05-27T01:28:16Z says:_ 

 That's a bug, but you also shouldn't be specifying "description:....".  Just do this:

```
task add ide
```

**Comment by taskwarrior** [comment]
_Sergey Trofimov on 2016-05-27T06:56:11Z says:_ 

 This syntax is used by the python library tasklib.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2016-05-27T12:50:06Z says:_ 

 Okay, but it's still wrong.

**Comment by taskwarrior** [comment]
_Tomas Babej on 2016-05-27T16:16:22Z says:_ 

 That's correct, currently tasklib does not treat description in any special manner when compared to other attributes.

**Comment by tbabej** [comment]
Fixed on 2.6.0.
